syntax/gitrebase.vim

```diff
@@ -33,6 +34,7 @@ hi def link gitrebaseSquash         Type
 hi def link gitrebaseFixup          Special
 hi def link gitrebaseExec           Function
 hi def link gitrebaseDrop           Comment
+hi def link gitrebaseBreak          PreProc
```

Comment:
Use `Macro`, as discussed.

Reply:
As far as I can tell, Macro is not defined by default, try `:h hl-Macro`

Reply:
See `:help group-name`.

Reply:
I see, didn't know that this is defined outside of the actual source. Will amend the commit

Reply:
done